### DISCUSSION TITLE
borderlands 2 is not giving me the cheats that I put down

### DISCUSSION THREAD

**Discussion by ethancurry6900** [bucanero/apollo-ps3#60] [Q&A] [answered]
When i tried to cheat borderlands it didn't work i looked at the cheats and i selected the ones that didn't have (not working) next to it so I don't know if it's just because my luck sucks or because i'm on ps3 hen...

**Comment by bucanero** [comment, marked as answer]
The issue is related to the special format of the save-game. A decrypted Borderlands 2 save is also compressed twice (another compression again after decompressed), and that's not currently supported by Apollo.

If you want to mod this save, you'll have to do it on a computer using Bruteforce Savedata, and the `offzip` tool to decompress the save file two times. (`offzip` parameter `-w 15`)

**Comment by ethancurry6900** [comment]
so I can't with apollo then my computer won't open bruteforce I tried it all ready and it won't open so thats why i turned to appolo... it is just confusing because borderlands 1 worked...

**Comment by ethancurry6900** [comment]
it reset my stats to deafult so like appolo will work but it resets my data
